scripts/turtles-dev.sh

```diff
@@ -13,18 +13,30 @@ kind create cluster --config "$BASEDIR/kind-cluster-with-extramounts.yaml"
 
 kubectl rollout status deployment coredns -n kube-system --timeout=90s
 
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.1/cert-manager.crds.yaml
-
-helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.5.1
+helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --create-namespace \
+    --version v1.12.3 \
+    --set installCRDs=true \
+    --wait
 
 kubectl rollout status deployment cert-manager -n cert-manager --timeout=90s
 
 export EXP_CLUSTER_RESOURCE_SET=true
+export CLUSTER_TOPOLOGY=true
 clusterctl init -i docker
 
-kubectl create namespace cattle-system
-
-helm install rancher rancher-stable/rancher --namespace cattle-system --set bootstrapPassword=admin --set replicas=1  --set hostname="$RANCHER_HOSTNAME" --set 'extraEnv[0].name=CATTLE_FEATURES' --set 'extraEnv[0].value=embedded-cluster-api=false' --set global.cattle.psp.enabled=false --version 2.7.5
+helm install rancher rancher-stable/rancher \
+    --namespace cattle-system \
+    --create-namespace \
```

Comment:
is this really needed for rancher installation?

Reply:
Do you mean creating a namespace?

Reply:
if we don't specify this flag the namespace has to be created with `kubectl`, we can delegate this task to helm, it's a common practice